Cargo.toml

```diff
@@ -29,12 +29,13 @@ error-chain = { version = "0.12.4", default-features = false }
 reqwest = { version = "0.11.4", features = ["blocking", "json"] }
```

Comment:
do you still need the 'blocking' feature?

Reply:
no, only tests use it and provided from someone year ago. I think that we can delete it safely (and remove from tests)

Reply:
I deleted this feature and fixed test